web/AdminLteAsset.php

```diff
@@ -43,6 +57,42 @@ public function init()
             $this->css[] = sprintf('css/skins/%s.min.css', $this->skin);
         }
 
+        // Append layout options
+        if ($layout = $this->layout) {
+            if (!array_key_exists($layout, AdminLteHelper::getLayoutHtmlClassOptions())) {
+                throw new InvalidConfigException('Invalid layout specified');
+            }
+	
+	        $layoutExtraAssetsOptions = self::getLayoutExtraAssetsOptions();
+            if (array_key_exists($layout, $layoutExtraAssetsOptions)) {
```

Comment:
This section could be problematic when bundling assets, since some stuff is loaded dynamically.

Might be an option to register the additional asset files manually via the `ExtraAsset...` - although this is not a perfect solution.

Reply:
But this stuff (extra assets) is needed by default for specific layout. Maybe, to do it more flexible we can create `AdminLteBowerAsset` that adds possibility for developers to connect scripts, styles from `bower_components` directory at `almasaeed2010/adminlte` package. 

But somehow logic that for specific `layout` options asset connect extra scripts, styles must be present.

Reply:
its related to https://github.com/dmstr/yii2-adminlte-asset/issues/180

Reply:
I think also solution can be write a notes to docs that some layouts by default adds extra js, css.

Reply:
But instead of setting a property, how about simply registering `AdminLteFixedLayoutAsset` instead of `AdminLteAsset`?